seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -181,16 +181,15 @@ angular.module('BE.seed.controller.menu', [])
           return;
         }
         if( !user_service.get_organization().id ){
-          var orgModalInstance = $uibModal.open({
+          $uibModal.open({
+            backdrop: 'static',
+            keyboard: false,
```

Comment:
Nice! I think that this'll limit the opportunities for weird edge cases to come up 👍